### ISSUE TITLE
Website responsiveness

### ISSUE THREAD

**Issue by AvaniVerma** [GirlScriptSummerOfCode/Kranti#15]
1. It's not entirely responsive between the width range 771 and 990
2. Check the warning message in console. There's a body tag in div
3. Increase the font size of 'Kranti' to make it significantly bigger than that of the nav contents, especially on bigger screen.
4. Make the logo a link that redirects to the home page.

To test please go to the link : https://codesandbox.io/s/github/amritsrivastava/Kranti/tree/amrit-patch-1

**Comment by devesh-verma** [comment]
I would like to take this one up. Plz assign me this

**Comment by AvaniVerma** [comment]
Sure @devshiva619 Go ahead

**Comment by devesh-verma** [comment]
Can u assign it to me?

**Comment by shreyateeza** [comment]
I would like to work on this issue if @devshiva619 won't work. :sweat_smile:

**Comment by ayush221b** [comment]
@AvaniVerma Is this issue still open? I would like to work on it. I have a fairly simple solution for the responsiveness problem and I can address the other points too.

**Comment by devesh-verma** [comment]
hey @ayush221b would you like to share your solution?

**Comment by ayush221b** [comment]
Yeah sure. Just replace `<nav className="navbar navbar-default navbar-fixed-top">` with `<nav className="navbar navbar-default navbar-static-top">` in Header.js

This should do the trick.

**Comment by devesh-verma** [comment]
@AvaniVerma This will be done by tonight. 
@ayush221b your approach was right, the only issue with that is the navbar becomes very broad.

**Comment by AvaniVerma** [comment]
@adi44 Would you like to work on this issue ?

**Comment by adi44** [comment]
yes

**Comment by AvaniVerma** [comment]
@adi44 Please work on it and tell us how much time you'll need.
